operators/deploy/custom-metrics-server/Chart.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: v2
+name: custom-metrics-server
+description: The CrownLabs CustomMetricsServer
```

Comment:
This Helm chart shall be included by the main one (in `deploy/crownlabs`, as well as the main values parameters shall be reported there.